src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -327,6 +327,7 @@
             "DSTOffsetListMaxSize"
         ],
         "Temperature Control": ["SupportedTemperatureLevels"],
+        "Laundry Dryer Controls": ["SupportedDrynessLevels"],
```

Comment:
Why is this here?  It's a list-typed attribute, right?  This JSON needs to list the non-list-typed attributes that are handled via AttributeAccessInterface.

Reply:
Okay, thanks for that, I have removed it in the next commit.